setup.cfg

```diff
@@ -20,7 +20,7 @@ classifiers =
 packages = find:
 install_requires =
     PyCifRW~=4.4
-    aiida-core>=2.1,<3
+    aiida-core[atomic-tools]>=2.1,<3
```

Comment:
Despite I originally wrote `atomic_tools`, the `setup-cfg-fmt` hook had a different opinion on what should it be. Let's see if it still works.

Reply:
I just removed this annoying hook.

Reply:
okay, let's remove it, I encountered it once in qeapp. But why you add these extra dependencies? Do we really need everything here? I think the `PyCifRW` is included in this extra, so if you add this, you need remove `PyCifRW` above. I personally disagree with adding these extra dependencies which include `pymatgen` which bring a lot of dependencies issue for other apps.